docs/test_the_installation.rst

```diff
@@ -92,17 +92,17 @@ Test the web interfaces
 
 #. Test that the source received the reply.
 
-   - Within Tor Browser, navigate back to the app-source-ths URL and
+   - Within Tor Browser, navigate back to the ``app-source-ths`` URL and
      use your previous test source codename to log in (or reload the
      page if it's still open) and check that the reply you just made
      is present.
 
 #. We highly recommend that you create persistent bookmarks for the
-   Source and Journalist Interface addresses within Tor Browser.
+   *Source and Journalist* Interface addresses within Tor Browser.
```

Comment:
I think this should say `*Source Interface* and *Journalist Interface* addresses` for consistency.